arch/posix/Kconfig

```diff
@@ -27,4 +27,15 @@ config ARCH_POSIX_RECOMMENDED_STACK_SIZE
 	  thread stack, the real stack is the native underlying pthread stack.
 	  Therefore the allocated stack can be limited to this size)
 
+config ARCH_POSIX_STOP_ON_FATAL_ERROR
+	bool "Terminate execution on fatal errors"
+	depends on ARCH_POSIX
+	default n
```

Comment:
`n` is the implicit default, so could skip the `default`. Makes things consistent too.

Reply:
Thanks Ulf :), corrected.